dbms/src/Server/Server.cpp

```diff
@@ -1231,6 +1231,8 @@ int Server::main(const std::vector<std::string> & /*args*/)
         LOG_INFO(log, "Global PageStorage run mode is {}", magic_enum::enum_name(global_context->getPageStorageRunMode()));
     }
 
+    global_context->getSharedContextDisagg()->initFastAddPeerContext();
```

Comment:
should it be init under `global_context->getSharedContextDisagg()->isDisaggregatedStorageMode()`?

Reply:
Yes, it should be here.